### ISSUE TITLE
reference to xml2js via git instead of https

### ISSUE THREAD

**Issue by dotob** [peerlibrary/node-xml4js#3]
hi, please make the dependency to xml2js as a https reference. behind a corporate firewall with proxy i cannot successfully download.

**Comment by mitar** [comment]
Can you explain a bit more what exactly would you like/need?

**Comment by mitar** [comment]
In title you are saying "instead of https", in description "as a https"? Which one now?

**Comment by dotob** [comment]
i could fix it with: git config --global url."https://".insteadOf git://

but what i mean is this part in package.json:

```javascript
"dependencies": {
    "xml2js": "git://github.com/peerlibrary/node-xml2js.git#peerlibrary",
```

it would be easier for user behind a proxy/firewall if you would use:

```javascript
"dependencies": {
    "xml2js": "https://github.com/peerlibrary/node-xml2js.git#peerlibrary",
```

**Comment by dotob** [comment]
sorry for tha bad title :-(

**Comment by mitar** [comment]
Can you make a pull request?

**Comment by mitar** [comment]
Fixed with #4.

**Comment by dotob** [comment]
thanks very much
